lib/diagnostic.js

```diff
@@ -17,7 +17,19 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+Description: Calender program for keeping track of things
+Entity: Event
+Attributes: when, where, duration
+Methods: conflictingEvent
```

Comment:
Just so that you think about it, `conflictingEvent` might need to take another Event as an argument to determine if they overlap. Also, it would probably be more conventional to call it `isConflictingEvent(blah)`, since it will presumably be returning a boolean.